File: Iniciante/URI 1035.cpp
#include <iostream>
 
int main() { 
    int a, b, c, d;

    scanf("%d %d %d %d", &a, &b, &c, &d);
    if (b > c && d > a && c > 0 && d > 0 && (c + d) > (a + b) && a % 2 == 0) {
        printf("Valores aceitos\n"); 
    } else { 
        printf("Valores nao aceitos\n");
    }

    return 0;
}

File: Iniciante/URI 1035.go
package main
 
import (
    "fmt"
)
 
func main() {
    var a, b, c, d int

    fmt.Scanf("%d", &a)
    fmt.Scanf("%d", &b)
    fmt.Scanf("%d", &c)
    fmt.Scanf("%d", &d)

	if (b > c && d > a && c > 0 && d > 0 && (c + d) > (a + b) && a % 2 == 0) {
		fmt.Println("Valores aceitos"); 
	} else { 
		fmt.Println("Valores nao aceitos");
	}	
}